src/subscription_manager/gui/managergui.py

```diff
@@ -523,7 +523,7 @@ def _getting_started_item_clicked(self, widget):
         try:
             # unfortunately, Gtk.show_uri does not work in RHEL 5
             DEVNULL = open(os.devnull, 'w')
-            subprocess.call(["gnome-open", "ghelp:subscription-manager"],
+            subprocess.call(["xdg-open", "ghelp:subscription-manager"],
```

Comment:
I modificated src/subscription_manager/ga_impls/ga_gtk2/Gtk.py, but gtk.show_uri() is not able to open documentation. Error from `/var/log/rhsm/rhsm.log`:

    2017-04-20 16:42:17,892 [WARNING] subscription-manager-gui:3450:MainThread @managergui.py:529 - Unable to open help documentation: Operation not supported